app/services/CreateBusinessDetailsService.scala

```diff
@@ -50,41 +51,52 @@ class CreateBusinessDetailsService @Inject()(val createIncomeSourceConnector: Cr
   def handleResponse(response: Either[CreateIncomeSourceErrorResponse, List[CreateIncomeSourceResponse]]): Future[Either[Error, CreateIncomeSourceResponse]] = {
     response match {
       case Right(List(incomeSourceId)) =>
-        Logger("application").info(s"[CreateBusinessDetailsService][createIncomeSource] - New income source created successfully: $incomeSourceId")
+        Logger("application").info(s"[CreateBusinessDetailsService][handleResponse] - New income source created successfully: $incomeSourceId")
         Future.successful(Right(incomeSourceId))
       case Right(_) =>
-        Logger("application").error("[CreateBusinessDetailsService][createIncomeSource] - failed to create, unexpected response")
+        Logger("application").error("[CreateBusinessDetailsService][handleResponse] - failed to create, unexpected response")
         Future.successful(Left(new Error("Failed to create incomeSources")))
       case Left(ex) =>
-        Logger("application").error("[CreateBusinessDetailsService][createIncomeSource] - failed to create")
+        Logger("application").error("[CreateBusinessDetailsService][handleResponse] - failed to create")
         Future.successful(Left(new Error(s"Failed to create incomeSources: $ex")))
     }
   }
 
   private def removeEmptyStrings(strOpt: Option[String]): Option[String] = {
-      strOpt match {
-        case Some(str) => if (str == "") None else Some(str)
-        case None => None
-      }
+    strOpt match {
```

Comment:
If this can be moved into extension method like (we can create story for this, as this is generic function we need across the project):

```
  1 implicit class OptionExtension(in: Option[String]) {
  2   def trim2(): Option[String] = {
  3      in match {
  4        case Some("") => None
  5        case _ => in
  6      }
  7   }
  8 }
  9
 10 val s : Option[String] = Some("")
 11
 12 println(s.trim2 )
```

Reply:
Next story created to cover this change: 
https://jira.tools.tax.service.gov.uk/browse/MISUV-6629